### PR TITLE
Improvements to the announcements cycle

### PR DESCRIPTION
**Pull request by Veldrovive**
I just added a little bit more functionality to make it more user friendly.

### CODE REVIEW

**Comment by HeroCC** [comment]
Neat! I'm not at a computer now, can you upload some screenshots?

**Comment by Veldrovive** [comment]
screen shot 2017-08-15 at 2 26 25 pm
screen shot 2017-08-15 at 2 23 09 pm

This is what they look like now, but the one with the pause doesn't fit on small screens. To pause right now you just hover anywhere over the container div. Another option is just to do it as pause on tap the entire div.

File: index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
   <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
   <script src="https://use.fontawesome.com/a26e812ff2.js"></script>
+  <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
```
Comment (HeroCC) [inline index.html]: Why do you need this? On the line above it imports fontawesome with some fancy metrics and CDN magic I don't understand or need.

**Comment by HeroCC** [comment]
Cool, does it work well on mobile? Can you make the buttons a shade lighter & perhaps a bit smaller? And they are hidden when the url is 'https://aspen.studenttech.tk/?billboard'? The billboards all have the '?billboard' at the end, and you can't click on a non interactive screen. And make sure the buttons / title are all the same size, the title area looks much smaller than the buttons.

Also, it would be neat if you had a little circle next to the announcement number that either filled or emptied (going a shade darker) as a timer for how much longer it will be displayed. Something like [this](http://git.wimbarelds.nl/TimeCircles/index.html) but without the text and much smaller? Or [this](http://www.html5canvastutorials.com/tutorials/html5-canvas-arcs/)? Not sure we can use a canvas though, IE doesn't support it on the billboards. You don't need to use that code (in fact I'd prefer something more lightweight / inhouse).

**Comment by HeroCC** [comment]
Also 10+ codacy triggers is far too many plz fix tanks ❤️

**Comment by Veldrovive** [comment]
I fixed the obvious issues that codacy had, but I didn't un-nest the functions. If you disagree with having nested functions in code, I'll gladly remove them, but they help me with understanding and debugging so I generally use them.

File: index.html (hunk 1 above)
Comment (Veldrovive) [reply index.html]: To be honest, I don't really know why it needs both, but it seems like one covers some symbols and the other covers the rest.

Comment (HeroCC) [reply index.html]: Any idea of what symbols? I am running 4.7, it looks like you're trying to import version 3.2?

Comment (Veldrovive) [reply index.html]: The original doesn't cover the arrows. I'll check 4.7 for arrows that would work as a substitute.

Comment (Veldrovive) [reply index.html]: Ok, it seems that they updated the name of the arrows so I fixed them and removed the import.

**Pull request merged**